package.json

```diff
@@ -19,7 +19,7 @@
     "eslint-plugin-import": "^2.14.0",
     "gh-pages": "^1.2.0",
     "husky": "^0.14.3",
-    "jest": "^22.4.4",
+    "jest": "^23.6.0",
```

Comment:
This broke a sentry unit test

```
 FAIL  modules/sentry/src/sentry.test.js
  ● Sentry.getUserUUID › When there is no Sentry.userUUID, sentry.getUserUUID return a different UUID each times it's called

    expect(received).not.toEqual(expected)

    Expected value to not equal:
      "524eeb96-8ef6-4a21-9a82-0f77dcc2f53e"
    Received:
      "524eeb96-8ef6-4a21-9a82-0f77dcc2f53e"
```

Reply:
Currently being resolved in https://github.com/wanadev/obsidianjs/pull/39